### PR TITLE
Add support for Drupal Console custom packages and languages

### PR DESCRIPTION
**Pull request by enzolutions**
The [Drupal Console](http://drupalconsole.com) project is working in complete some goals to release his first 1.0.0 stable release more info at [https://drupalconsole.com/articles/help-us-complete-drupal-console-stable-release](https://drupalconsole.com/articles/help-us-complete-drupal-console-stable-release)

One of the main component required for that its provide a way to Drupal developer to be able to add custom packages of commands to be used in their Drupal installation, independent of contributed modules command 

Another goal for this release its allows the usage of custom languages, that will reduce the size of Drupal Console installer and people will be freer in terms of their contribution and how to use the Drupal Console project.

In order to warranty this flexibility is required the inclusion of two new composers installers types **drupal-console** for packages and **drupal-console-languages** for languages.

### CODE REVIEW

**Comment by darol100** [comment]
👍  For this PR.

**Comment by david-garcia-garcia** [comment]
Drupal console will become the defacto command line management tool for Drupal. This is very important for supporting the Drupal project.

**Comment by ofry** [comment]
This PR has merge conflicts.

**Comment by enzolutions** [comment]
Thanks @ofry , I just updated the PR

File: src/Composer/Installers/DrupalInstaller.php [hunk 1]
```diff
@@ -14,5 +14,7 @@ class DrupalInstaller extends BaseInstaller
         'custom-module'    => 'modules/custom/{$name}/',
         'custom-profile'   => 'profiles/custom/{$name}/',
         'drupal-multisite' => 'sites/{$name}/'
```
Comment (alcohol) [inline src/Composer/Installers/DrupalInstaller.php]: Missing trailing `,` here

**Comment by enzolutions** [comment]
@alcohol travis is failing, could due to the latest change?

**Comment by alcohol** [comment]
@enzolutions no I already pointed out why it fails, see my comment about the missing trailing comma.

File: src/Composer/Installers/DrupalInstaller.php (hunk 1 above)
Comment (enzolutions) [reply src/Composer/Installers/DrupalInstaller.php]: @alcohol Thank for your review, I fixed the issue.

**Pull request merged**